pkg/types.go

```diff
@@ -182,3 +299,61 @@ func FilePathToExtension(path string) string {
 
 	return path[dotIndex+1:]
 }
+
+func WrapSpecInK8s(data []byte) ([]byte, error) {
```

Comment:
This function name is a bit confusing.

Reply:
Yeah, I'm trying to come up with a better name. 

Since we only store the `spec` part of the manifest, this part wraps it so we have 
```{
   manifest: {},
   spec: {...}
}```

instead of just the `spec` contents.